CharacterGenerator/CharacterGenerator/Soldier.h

```diff
@@ -0,0 +1,26 @@
+//
+//  Soldier.h
+//  CharacterGenerator
+//
+//  Created by Z on 8/2/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface Soldier : NSObject
+
+@property (nonatomic) NSString *name;
+
+@property (nonatomic) NSString *primaryWeapon;
```

Comment:
This looks like another situation where using a custom model class might make things more modular. If every soldier has a `primaryWeapon` and `primaryWeaponType` that could probably be abstracted into a `PrimaryWeapon` class with `name` and `type` properties. Then each soldier would have a instance of that class.